application-xpoll-ui/src/main/resources/XPoll/XPollSheet.xml

```diff
@@ -52,7 +52,7 @@
   #set ($userReference = $services.model.resolveDocument($userName))
   #set ($userDoc = $xwiki.getDocumentAsAuthor($userReference))
   {{html wiki='true' clean='false'}}
-    &lt;div class='clearfix user'&gt;
+    &lt;div class='clearfix xpollUser'&gt;
```

Comment:
Doesn't the ``user`` class have any styles associated with it? I know this class is used by https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/macros.vm#L246 (maybe we can reuse this macro?). It might be safer to keep the ``user`` class and add another one to make it a user specific to Poll Application, e.g. ``user xpollUser``.